### PR TITLE
Provide aria labels for buttons

### PR DESCRIPTION
**Pull request by bikegriffith**
The most basic thing possible to start to appease #863

### CODE REVIEW

File: src/plugins/dvr_controls/public/index.html [hunk 1]
```diff
@@ -1,2 +1,2 @@
 <div class="live-info"><%= live %></div>
-<button type="button" class="live-button"><%= backToLive %></button>
+<button type="button" class="live-button" aria-label="Back to live"><%= backToLive %></button>
```
Comment (leandromoreira) [inline src/plugins/dvr_controls/public/index.html]: The names are i18n, you shouldn't replace them.

**Comment by leandromoreira** [comment]
Thanks @bikegriffith I'd also like to point that one test stop working  https://travis-ci.org/clappr/clappr/builds/173304206

```
Chromium 37.0.2062 (Ubuntu 0.0.0) UIObject selects elements within the component FAILED
	Error: timeout of 2000ms exceeded. Ensure the done() callback is being called in this test.
Chromium 37.0.2062 (Ubuntu 0.0.0): Executed 50 of 179 (1 FAILED) (0 secs / 18.463 secs)
```

File: src/plugins/dvr_controls/public/index.html (hunk 1 above)
Comment (towerz) [inline src/plugins/dvr_controls/public/index.html]: Agree, instead of using a literal string "Back to Live", you should use the value of `backToLive`.

**Comment by towerz** [comment]
Thanks @bikegriffith! 🍻

**Pull request merged**